# generated by patch-package 6.4.8 on 2022-01-30 14:56:40
#
# command:
#   npx patch-package yoga-layout-prebuilt
#
# declared package:
#   yoga-layout-prebuilt: ^1.10.0
#
diff --git a/node_modules/yoga-layout-prebuilt/yoga-layout/build/Release/nbind.js b/node_modules/yoga-layout-prebuilt/yoga-layout/build/Release/nbind.js
index 0853309..3ddc263 100644
--- a/node_modules/yoga-layout-prebuilt/yoga-layout/build/Release/nbind.js
+++ b/node_modules/yoga-layout-prebuilt/yoga-layout/build/Release/nbind.js
@@ -1143,7 +1143,9 @@
       }
     }_nbind.addMethod = addMethod;function throwError(message) {
       throw new Error(message);
-    }_nbind.throwError = throwError;_nbind.bigEndian = false;_a = _typeModule(_typeModule), _nbind.Type = _a.Type, _nbind.makeType = _a.makeType, _nbind.getComplexType = _a.getComplexType, _nbind.structureList = _a.structureList;var BindType = function (_super) {
+    }
+// https://github.com/vadimdemedes/yoga-layout-prebuilt/pull/18
+var _a;_nbind.throwError = throwError;_nbind.bigEndian = false;_a = _typeModule(_typeModule), _nbind.Type = _a.Type, _nbind.makeType = _a.makeType, _nbind.getComplexType = _a.getComplexType, _nbind.structureList = _a.structureList;var BindType = function (_super) {
       __extends(BindType, _super);function BindType() {
         var _this = _super !== null && _super.apply(this, arguments) || this;_this.heap = HEAPU32;_this.ptrSize = 4;return _this;
       }BindType.prototype.needsWireRead = function (policyTbl) {
